example/shared_bindings/README.md

```diff
@@ -0,0 +1,10 @@
+# Shared bindings
+
+An example to showcase how bindings can share types
+with other bindings
+
+## Generating bindings
+At the root of this example (`example/shared_bindings`), run -
+```
+./run.sh
```

Comment:
You could consider making this a Dart script (with `Process.run`s) instead of a shell script.

Reply:
Done